trackintel/geogr/distances.py

```diff
@@ -161,4 +162,17 @@ def meters_to_decimal_degrees(meters, latitude):
     float
         An approximation of a distance (given in meters) in degrees.
     """
-    return meters / (111.32 * 1000.0 * cos(latitude * (pi / 180.0)))
\ No newline at end of file
+    return meters / (111.32 * 1000.0 * cos(latitude * (pi / 180.0)))
+
+
+def check_crs(data):
```

Comment:
- Missing docstring
- I think the function name should be more specific and that a "check" function should not throw errors or warnings. At the moment the function is very specific and it is unlikely that we will find another use-case that needs exactly this behavior. So we could either make the function more general (e.g., check_if_wgs84) or move this check directly into the calculate length function.

Reply:
ok, I will move this functionality back to the length function. I only placed in geogr.distances because I noted this from our last meeting. But I agree, that it makes only sense if it is a general function.

Reply:
Yeah... sorry for the back and forth... In the meeting it sounded like a good idea 🤷‍♂️